lib/utils/bedroomCalculator.ts

```diff
@@ -0,0 +1,60 @@
+function calculateBedrooms(...people: Array<any>) {
+  // one bedroom each...unless it's a couple
+  let relationshipType = people[0].pop();
```

Comment:
Try to avoid using `let`. Also, `pop` is a mutation which should also be avoided.